deploy/traits.yaml

```diff
@@ -820,8 +820,17 @@ traits:
   platform: false
   profiles:
   - OpenShift
-  description: The Route trait can be used to configure the creation of OpenShift
-    routes for the integration.
+  description: 'The Route trait can be used to configure the creation of OpenShift
+    routes for the integration. The certificate and key contents may be stored either
```

Comment:
I'd suggest: "may be source either from"

Reply:
suggestion accepted.